src/fswAlgorithms/imageProcessing/opticalFlow/opticalFlow.cpp

```diff
@@ -209,6 +209,30 @@ void OpticalFlow::makeMask (cv::Mat const &inputBWImage, cv::Mat &mask) const {
     cv::Mat erosionKernel = cv::Mat::ones(this->limbMask, this->limbMask, CV_8U);
     cv::erode(thresholded, thresholded, erosionKernel);
     thresholded.convertTo(mask, CV_8UC1);
+
+    /*! Mask the edges of the camera FOV in order to avoid effects of camera distortions.
+     * Point in opencv is column, row */
+    /*! - Left edge removal */
+    cv::rectangle(mask, cv::Point(0,0),
```

Comment:
:large_blue_circle: I'm a fan of a parameter per new line. I think it makes the list easy to scan when many params. Just a thought, but of course your call if there's a reason not to. E.g. 

```
cv::rectangle(mask,
              cv::Point(0,0),
              cv::Point(this->limbMask, mask.size().height),
              cv::Scalar(0),
              -1);
```

Reply:
Agreed!